Ryan-D-Miller/RefactorTractor
# Begin retrievelistByNameOrIngredients function

### Is this a fix or a feature?
 - Feature
### What does it fix/do?
 - Began work on retrieveListByNameOrIngreidents function. Still need to add the ingredient functionality to the method
### Where should the reviewer start?
 - recipeRepository.js
### How should this be tested?
 - Viewing the comp in the browser to make sure ingredients are filtered in the search bar
